content/support-training.html

```diff
@@ -90,6 +90,13 @@ <h2>Commercial support</h2>
               </a>
           </div>
 
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
+
+            <a href="https://sighup.io/" class="btn btn-training btn-default btn-large" target="_blank">
```

Comment:
Is there a connection to Prometheus support here that could be clarified through a more specific link? From just looking at the homepage, it seems all about a Kubernetes distribution / Kubernetes management.

I realize some others in this list are also a bit unspecific in their links, but I'm wondering whether we should ask for / require more Prometheus-focussed info, since this is what people looking at this page will be wondering about as well.